saheedt/trans-far
# Ft create uploads 164363866

#### What does this PR do?
- Add ability to upload files
#### Description of Task to be completed?
- Upload files with title and description
- Store uploaded file(s) in Amazon s3 bucket via active storage
#### How should this be manually tested?
- Navigate to http://localhost/uploads/new
- select file(s) and add optional title and description
- Click upload button
- On successful upload, page should navigate to http://localhost/uploads/show
#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
- [#164363866](https://www.pivotaltracker.com/story/show/164363866)
#### Screenshots (if appropriate)
- N/A
#### Questions:
- N/A
